.github/workflows/linters.yml

```diff
@@ -29,12 +29,12 @@ jobs:
       - name: Setup Webhint
         run: |
           npm install --save-dev hint@7.x
-          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
```

Comment:
- [ ] It looks like you forgot to set up the eslint checks on GitHub. Normally, the linter checks should be 5 but now they are four. That means eslint action check is not set up 😃. But don't worry, it is just a second to finish it 😎. Please update your `linters.yml` file by using this one 👉 [HERE](https://github.com/microverseinc/linters-config/tree/master/html-css-js/.github/workflows) because it includes  all the rules to run all 5 necessary linter checks for this project 😉